frontend/src/pages/Overview/reducer.js

```diff
@@ -54,7 +60,7 @@ const defaultState = fromJS({
   idForInfo: "",
   isProjectAdditionalDataShown: false,
   searchTerms: [],
-  projectView: "card"
+  projectView: ""
```

Comment:
why? Default view should be set to either card or table

Reply:
done